docs/tutorials/google/start.ipynb

```diff
@@ -1,478 +1,532 @@
 {
```

Comment:
<p>s/After running the block you will see/After running the block, you will see</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/6342/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>